PSSwagger/PSSwagger.psm1

```diff
@@ -238,7 +240,15 @@ function New-PSSwaggerModule {
         [Parameter(Mandatory = $false, ParameterSetName = 'SpecificationPath')]
         [Parameter(Mandatory = $false, ParameterSetName = 'SpecificationUri')]
         [switch]
-        $ConfirmBootstrap
+        $ConfirmBootstrap,
+
+        [Parameter(Mandatory = $false, ParameterSetName = 'SpecificationPath')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'SpecificationUri')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'SdkAssemblyWithSpecificationPath')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'SdkAssemblyWithSpecificationUri')]
+        [string]
+        [ValidateSet('None', 'PSScriptAnalyzer')]
+        $Formatter = 'None'
```

Comment:
Consider conveying this in the parameter description or in the cmdlet description. So that user knows that parameters on this cmdlet override the config values.

---
In reply to: [153916061](https://github.com/PowerShell/PSSwagger/pull/381#discussion_r153916061) [](ancestors = 153916061)